client/db/src/lib.rs

```diff
@@ -1242,13 +1247,23 @@ impl<Block: BlockT<Hash=H256>> Backend<Block> {
 
 			let finalized = if operation.commit_state {
 				let mut changeset: sc_state_db::ChangeSet<Vec<u8>> = sc_state_db::ChangeSet::default();
+				let mut ops: u64 = 0;
+				let mut bytes: u64 = 0;
 				for (key, (val, rc)) in operation.db_updates.drain() {
 					if rc > 0 {
+						ops += 1;
+						bytes += key.len() as u64 + val.len() as u64;
+
```

Comment:
I feel there are too many `u64`. But maybe is me :).

Reply:
theoretically they can overflow when added before conversion, so it is a safer construct :)